ludwig/schema/utils.py

```diff
@@ -577,6 +578,31 @@ def IntegerRange(
     )
 
 
+@DeveloperAPI
+def Float(default: Union[None, float], allow_none=False, description="", parameter_metadata: ParameterMetadata = None):
+    """Returns a dataclass field with marshmallow metadata strictly enforcing (non-float) inputs."""
```

Comment:
Do you mean strictly enforcing float inputs?

Reply:
Ah, thought I fixed that. Thanks for marking it.